tests/framework/e2e/etcdctl.go

```diff
@@ -337,8 +337,8 @@ func (ctl *EtcdctlV3) HashKV(ctx context.Context, rev int64) ([]*clientv3.HashKV
 		return nil, err
 	}
 	resp := make([]*clientv3.HashKVResponse, len(epHashKVs))
-	for _, e := range epHashKVs {
-		resp = append(resp, e.HashKV)
+	for i, e := range epHashKVs {
+		resp[i] = e.HashKV
```

Comment:
Good catch.  Please submit a separate PR to resolve this so that we can get it merged firstly.

Reply:
Ok, I'll create 2 two separate PRs for this change and for AuthEnable/Disable one.

Reply:
https://github.com/etcd-io/etcd/pull/14607 and https://github.com/etcd-io/etcd/pull/14608